goth_tests/goth-config.yml

```diff
@@ -20,6 +20,12 @@ docker-compose:
     # commit-hash: ...
     # release-tag: ...
 
+    # binary-path: "/home/przemek/Code/Golem/yagna/target/debug/bin"
+    # deb-path: ...
+    # branch: "2466_outbound_partner_rule_e2e"
+    # commit-hash: "7d66a8ed42e28c5cbe977793889157f7682faca5"
+    release-tag: "v0.13.0-raw-rc3"
```

Comment:
Would this overwrite the yagna version used in goth tests? I think goth tests in yagna repository should use the version that is build in a related CI step.

If it does not affect the yagna version used then I am confused what is the purpose of this flag.

Reply:
Fixed it a moment ago.